### PR TITLE
Improve database performance

### PR DESCRIPTION
**Pull request by AbayomiOlaoye**
### Summary of Pull Request

We implemented the following project requirements:
- [x] The following queries are taking too much time (1 sec = 1000ms can be considered as too much time for database query). Try them on your machine to confirm it:

    SELECT COUNT(*) FROM visits where animal_id = 4;
    SELECT * FROM visits where vet_id = 2;
    SELECT * FROM owners where email = 'owner_18327@mail.com';

- [x] Use EXPLAIN ANALYZE on the previous queries to check what is happening. Take screenshots of them - they will be necessary later.
- 
- 
- 

- [x] Find a way to decrease the execution time of the first query. Look for hints in the previous lessons.
- 

- [x] Find a way to improve execution time of the other two queries

- 
-

### CODE REVIEW

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive pull request message ✔️
- Good usage of gitflow ✔️
almost

## Required Changes ♻️
- [ ] As request in the project requirements, let's have a pair of screenshots in your pull request. The first screenshot should show how much time the query takes BEFORE the improvements introduced, the second should show how much time the query takes AFTER. In total you should have six screenshots.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql [hunk 1]
```diff
@@ -70,3 +70,9 @@ INSERT INTO visits (animals_id, vets_id, date_of_visit)
 	(9, 2, '2020-08-03'),
 	(10, 3, '2020-05-24'),
 	(10, 1, '2021-01-11');
+
+-- This will add 3.594.280 visits considering you have 10 animals, 4 vets, and it will use around ~87.000 timestamps (~4min approx.)
```
Comment (HAMMAS-SALEEM) [inline data.sql:74]: **[Optional]**

- [ ] This is a pair programming project but i cannot see any commit from your partner side. Kindly follow the requirements :blush:

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

### Hey @AbayomiOlaoye 👋

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (AbayomiOlaoye) [reply data.sql:74]: Thank you, we shall fix that in the next request.

**Pull request merged**